README.md

```diff
@@ -17,6 +17,7 @@
 [![Visual Studio Marketplace Version](https://img.shields.io/visual-studio-marketplace/v/tonybaloney.vscode-pets?color=blue&logo=visual-studio)](https://marketplace.visualstudio.com/items?itemName=tonybaloney.vscode-pets&WT.mc_id=python-17801-anthonyshaw)
 [![Visual Studio Marketplace Installs](https://img.shields.io/visual-studio-marketplace/i/tonybaloney.vscode-pets?logo=visualstudio)](https://marketplace.visualstudio.com/items?itemName=tonybaloney.vscode-pets&WT.mc_id=python-17801-anthonyshaw)
 [![Visual Studio Marketplace Downloads](https://img.shields.io/visual-studio-marketplace/d/tonybaloney.vscode-pets?logo=visualstudio)](https://marketplace.visualstudio.com/items?itemName=tonybaloney.vscode-pets&WT.mc_id=python-17801-anthonyshaw)
+[![Crowdin](https://badges.crowdin.net/vscode-pets/localized.svg)](https://crowdin.com/project/vscode-pets)
```

Comment:
Vs code won't allow this to be bundled in the package (see the build error)

Reply:
Got it, so I will remove this badge, thank you!

There is another way to add a localized badge - [Badges & Status Images](https://store.crowdin.com/bds)

You can install this app and then add the shields-based badge if you don't mind

Or, at least, add the Crowdin project link to the Readme